components/script/dom/identityhub.rs

```diff
@@ -77,6 +79,10 @@ impl IdentityHub {
     fn create_command_encoder_id(&mut self) -> CommandEncoderId {
         self.command_encoders.alloc(self.backend)
     }
+
+    fn create_sampler_id(&mut self) -> SamplerId {
```

Comment:
We shouldn't need these helper methods. The caller is `Identities::crate_sampler_id`, and it can do:
```rust
self.select(backend).samplers.alloc(backend)
```
Moreover, we may not even need `IdentityHubb::backend` field